libr/util/w32-sys.c

```diff
@@ -144,13 +131,15 @@ char *ReadFromPipe(HANDLE fh) {
 	str = malloc (strsz);
 	for (;;) {
 		bSuccess = ReadFile (fh, chBuf, BUFSIZE, &dwRead, NULL);
-		if (! bSuccess || dwRead == 0) break;
-		chBuf[dwRead] = '\0';
```

Comment:
There is no need to add a null byte since no string related functions are applied to it. (Only memcpy)